src/freenet/pluginmanager/PluginManager.java

```diff
@@ -942,6 +952,27 @@ public FredPluginFCP getFCPPlugin(String plugname) {
 		return null;
 	}
 
+    /**
+     * Get the {@link FredPluginFCPMessageHandler.ServerSideFCPMessageHandler} of the plugin with
+     * the given class name.
+     * 
+     * @param pluginClassName
+     *            See {@link #getPluginInfoByClassName(String)}.
+     * @throws PluginNotFoundException
+     *             If the specified plugin is not loaded or does not provide an FCP server.
+     */
+    public FredPluginFCPMessageHandler.ServerSideFCPMessageHandler
+            getPluginFCPServer(String pluginClassName)
+                throws PluginNotFoundException{
+        
+        PluginInfoWrapper piw = getPluginInfoByClassName(pluginClassName);
+        if(piw != null && piw.isFCPServerPlugin()) {
+            return (FredPluginFCPMessageHandler.ServerSideFCPMessageHandler)piw.plug;
```

Comment:
Fixed by commit 6e00f172fdebc4e11fbeab9a0f5dad01351e6c44 - which also is a good preparation of what I suggested at https://github.com/freenet/fred/pull/319/files#r22537901 (= to allow having the plugin interfaces be implemented by different classes than the plugin main class)